server/controllers/RequestController.js

```diff
@@ -17,6 +17,39 @@ export const getOneRequest = async (req, res) => {
   }
 };
 
+export const getRequestsWithin3Months = async (req, res) => {
+  try {
+    // The month goes from 0 to 11
+    let startMonth;
+    let endMonth;
+    const thisMonth = (new Date()).getMonth();
+    const thisYear = (new Date()).getFullYear();
+
+    for (startMonth = 0; startMonth <= 9; startMonth += 3) {
+      endMonth = startMonth + 2;
+      if (startMonth <= thisMonth && thisMonth <= endMonth) {
+        break;
+      }
+    }
+
+    const startDate = new Date(thisYear, startMonth, 1);
+    const endDate = new Date(new Date(thisYear, endMonth + 1, 1) - 1);
```

Comment:
Can you have some in-line comment to make it understandable?

Reply:
Done